getting-started/templates/systemlink-values.yaml

```diff
@@ -64,6 +64,22 @@ global:
     ##
     includeUserInfo: false
 
+## Core configuration for the RabbitMQ message broker
+##
+rabbitmq:
+  ## Policy used whend starting pods in the stateful set.
+  # <ATTENTION> - RabbitMQ pods must be started in reverse order of how they were shut down.
+  #               After an unexpected shutdown of the cluster, temporarily set podManagementPolicy
+  #               to "Parallel" and forceBoot to true to restore the cluster to a healthy state.
```

Comment:
```suggestion
  #               to "Parallel" and forceBoot to true.
```

Reply:
Applying this manually to fixup line breaks.